### PR TITLE
validation: multiple-fixes-1

### PR DESCRIPTION
**Pull request by MelanieHom**

### CODE REVIEW

**Comment by MelanieHom** [comment]
@Lauragra 

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

File: concepts/data-connect-quickstart-05.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 ---
+title: Microsoft Graph Data Connect tutorial - Using Azure Data Factory to extract data
+description: Learn how to create an Azure Data Factory pipeline using Microsoft Graph data connect to extract data from Microsoft 365 to an Azure Storage account
```
Comment (jasonjoh) [inline concepts/data-connect-quickstart-05.md:3]: This change should be reverted. These are include files for a tutorial. The H1 you added is duplicated (for example, see https://review.docs.microsoft.com/en-us/graph/data-connect-quickstart?branch=pr-en-us-16619&tabs=Microsoft365&tutorial-step=6)

File: concepts/data-connect-quickstart-06.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
 ---
+title: Microsoft Graph Data Connect tutorial - Managing requests with PowerShell and the Microsoft 365 Admin Center
```
Comment (jasonjoh) [inline concepts/data-connect-quickstart-06.md:2]: This change should be reverted. These are include files for a tutorial. The H1 you added is duplicated (for example, see https://review.docs.microsoft.com/en-us/graph/data-connect-quickstart?branch=pr-en-us-16619&tabs=Microsoft365&tutorial-step=6)

File: concepts/data-connect-quickstart-07.md [hunk 3]
```diff
@@ -1,7 +1,9 @@
 ---
+title: Microsoft Graph Data Connect tutorial - Building an ASP.NET project
```
Comment (jasonjoh) [inline concepts/data-connect-quickstart-07.md:2]: This change should be reverted. These are include files for a tutorial. The H1 you added is duplicated (for example, see https://review.docs.microsoft.com/en-us/graph/data-connect-quickstart?branch=pr-en-us-16619&tabs=Microsoft365&tutorial-step=6)

File: concepts/data-connect-quickstart-08.md [hunk 4]
```diff
@@ -1,7 +1,9 @@
 ---
+title: Microsoft Graph Data Connect tutorial - Complete
```
Comment (jasonjoh) [inline concepts/data-connect-quickstart-08.md:2]: This change should be reverted. These are include files for a tutorial. The H1 you added is duplicated (for example, see https://review.docs.microsoft.com/en-us/graph/data-connect-quickstart?branch=pr-en-us-16619&tabs=Microsoft365&tutorial-step=6)

**Review by jasonjoh — CHANGES_REQUESTED** [review]

File: concepts/data-connect-quickstart-05.md (hunk 1 above)
Comment (MelanieHom) [reply concepts/data-connect-quickstart-05.md:3]: H1 removed and metadata updated.

**Comment by MelanieHom** [comment]
> Those tutorial topics are still broken:
> 
> image
> 
> I think the `title` metdata may be causing this. It's not needed in an include file.

The title metadata has been removed. Let me know if this is okay before I do the same with the other PRs. Thanks!

**Comment by jasonjoh** [comment]
Well, I don't know why, but it's still showing double titles. You can check it here: https://review.docs.microsoft.com/en-us/graph/data-connect-quickstart?branch=pr-en-us-16619&tabs=Microsoft365&tutorial-step=6

You might want to just revert all your changes to the quickstart files. I'm pretty sure those descriptions don't get included in the built HTML - they come from the YAML file.

**Comment by jasonjoh** [comment]
If you're trying to resolve validation warnings in the include files, a better fix is to move them into an "includes" subfolder. A similar fix was done for the connector tutorial in https://github.com/microsoftgraph/microsoft-graph-docs/pull/16621.

**Review by jasonjoh — APPROVED** [review]

**Pull request merged**